README.rst

```diff
@@ -31,7 +31,7 @@ Initial Setup: steps and confirmation
 
 Note: this setup process is temporary; we will be working on a Tutor plugin
 
-.. code-block::
+.. code-block:: shell
```

Comment:
@pshiu: Curious if you get the same error with `none` or `text`?

Reply:
Good thought. Unfortunately, I still do.

<details>
<summary>Output for code-block:: none</summary>

```
(cc) commerce-coordinator % git diff
diff --git a/README.rst b/README.rst
index cb0514d..a21993e 100644
--- a/README.rst
+++ b/README.rst
@@ -31,7 +31,7 @@ Initial Setup: steps and confirmation
 
 Note: this setup process is temporary; we will be working on a Tutor plugin
 
-.. code-block:: shell
+.. code-block:: none
 
   # Clone the repository
   git clone git@github.com:edx/commerce-coordinator.git
@@ -74,7 +74,7 @@ Note: this setup process is temporary; we will be working on a Tutor plugin
 
 Every time you develop something in this repo
 =============================================
-.. code-block:: shell
+.. code-block:: none
 
   # Grab the latest code
   git checkout main
@@ -104,7 +104,7 @@ Every time you develop something in this repo
 
 Local testing with Celery
 =========================
-.. code-block:: shell
+.. code-block:: none
 
   # Start redis in devstack from your local devstack directory
   make dev.up.redis
(cc) commerce-coordinator % rstcheck -r .

An `AttributeError` error occured. This is most propably due to a code block directive (code/code-block/sourcecode) without a specified language. This may result in a false negative for source: 'docs/readme.rst'. See https://rstcheck-core.readthedocs.io/en/latest/faq/#code-blocks-without-language-sphinx for more information.
An `AttributeError` error occured. This is most propably due to a code block directive (code/code-block/sourcecode) without a specified language. This may result in a false negative for source: 'docs/changelog.rst'. See https://rstcheck-core.readthedocs.io/en/latest/faq/#code-blocks-without-language-sphinx for more information.
README.rst:3: (INFO/1) Duplicate implicit target name: "how to contribute".
README.rst:169: (INFO/1) Duplicate implicit target name: "getting help".
CHANGELOG.rst:35: (INFO/1) Duplicate implicit target name: "added".
docs/testing.rst:1: (INFO/1) Hyperlink target "chapter-testing" is not referenced.
docs/features.rst:1: (INFO/1) Hyperlink target "chapter-features" is not referenced.
docs/decisions.rst:6: (INFO/1) Hyperlink target "adrs" is not referenced.
docs/internationalization.rst:1: (INFO/1) Hyperlink target "chapter-i18n" is not referenced.
Error! Issues detected.
```
</details>

<details>
<summary>Output for code-block:: text</summary>

```
(cc) commerce-coordinator % git diff
diff --git a/README.rst b/README.rst
index cb0514d..1ee0354 100644
--- a/README.rst
+++ b/README.rst
@@ -31,7 +31,7 @@ Initial Setup: steps and confirmation
 
 Note: this setup process is temporary; we will be working on a Tutor plugin
 
-.. code-block:: shell
+.. code-block:: text
 
   # Clone the repository
   git clone git@github.com:edx/commerce-coordinator.git
@@ -74,7 +74,7 @@ Note: this setup process is temporary; we will be working on a Tutor plugin
 
 Every time you develop something in this repo
 =============================================
-.. code-block:: shell
+.. code-block:: text
 
   # Grab the latest code
   git checkout main
@@ -104,7 +104,7 @@ Every time you develop something in this repo
 
 Local testing with Celery
 =========================
-.. code-block:: shell
+.. code-block:: text
 
   # Start redis in devstack from your local devstack directory
   make dev.up.redis
(cc) commerce-coordinator % rstcheck -r . 

An `AttributeError` error occured. This is most propably due to a code block directive (code/code-block/sourcecode) without a specified language. This may result in a false negative for source: 'docs/readme.rst'. See https://rstcheck-core.readthedocs.io/en/latest/faq/#code-blocks-without-language-sphinx for more information.
An `AttributeError` error occured. This is most propably due to a code block directive (code/code-block/sourcecode) without a specified language. This may result in a false negative for source: 'docs/changelog.rst'. See https://rstcheck-core.readthedocs.io/en/latest/faq/#code-blocks-without-language-sphinx for more information.
README.rst:3: (INFO/1) Duplicate implicit target name: "how to contribute".
README.rst:169: (INFO/1) Duplicate implicit target name: "getting help".
CHANGELOG.rst:35: (INFO/1) Duplicate implicit target name: "added".
docs/testing.rst:1: (INFO/1) Hyperlink target "chapter-testing" is not referenced.
docs/features.rst:1: (INFO/1) Hyperlink target "chapter-features" is not referenced.
docs/decisions.rst:6: (INFO/1) Hyperlink target "adrs" is not referenced.
docs/internationalization.rst:1: (INFO/1) Hyperlink target "chapter-i18n" is not referenced.
Error! Issues detected.

```
</details>